src/components/molecules/videoCard/VideoCard.tsx

```diff
@@ -48,7 +50,29 @@ export function VideoCard({ video, pageType }: VideoCardProps) {
   return (
     <Card>
       <CardMedia>
-        <PreModalImage videoInfo={video} />
+        <div style={{ position: 'relative' }}>
+          <PreModalImage videoInfo={video} />
+          <FaCircle
+            style={{
+              position: 'absolute',
+              top: '50%',
+              left: '50%',
+              transform: 'translate(-50%, -50%)',
+              fontSize: '48px',
+              color: 'rgba(105, 105, 105, 0.75)',
+            }}
+          />
+          <BiSolidRightArrow
+            style={{
+              position: 'absolute',
+              top: '50%',
+              left: '50%',
+              transform: 'translate(-50%, -50%)',
+              fontSize: '30px',
+              color: '#303030',
+            }}
+          />
```

Comment:
🔧 Just based on the image, Vika's play button is a bit more angular. 

Can we do `GrPlayFill` or `IoPlaySharp`?

Reply:
This is `GrPlayFill`:
<img width="436" alt="Screen Shot 2024-04-03 at 2 56 30 PM" src="https://github.com/mwkwsd/sensensomething/assets/102488171/5d9d30bb-31a1-4118-9aa1-db5d5a73623e">

This is `IoPlaySharp`:
<img width="438" alt="Screen Shot 2024-04-03 at 2 57 53 PM" src="https://github.com/mwkwsd/sensensomething/assets/102488171/7d40e25a-dff6-48c0-83ab-0775de41f901">

It looks closer to `GrPlayFill` to me, but let me know on preference!

Reply:
Probably `GrPlayFill`